app/src/main/java/com/kickstarter/ui/viewholders/ProjectCardViewHolder.java

```diff
@@ -118,10 +120,30 @@ public ProjectCardViewHolder(final @NonNull View view, final @NonNull Delegate d
       .compose(observeForUI())
       .subscribe(ViewUtils.setGone(this.featuredViewGroup));
 
-    this.viewModel.outputs.friendAvatarUrl()
+    this.viewModel.outputs.friendAvatar2IsHidden()
```

Comment:
can rename these outputs from `IsHidden` to `IsGone` to more properly reflect what's happening in the view!